src/acquisition/covid_hosp/state_daily/database.py

```diff
@@ -0,0 +1,84 @@
+# first party
+from delphi.epidata.acquisition.covid_hosp.common.database import Database as BaseDatabase
+from delphi.epidata.acquisition.covid_hosp.common.utils import Utils
+
+
+class Database(BaseDatabase):
+
+  TABLE_NAME = 'covid_hosp_state_timeseries'
```

Comment:
```suggestion
  # note we share a database with state_timeseries
  TABLE_NAME = 'covid_hosp_state_timeseries'
```

& add a matching comment in `state_timeseries/database.py`

Reply:
just noting that i recently changed the table name from `covid_hosp` to `covid_hosp_state_timeseries` since i thought each dataset would have a separate table. since we're sharing tables, i'm going to suggest renaming that back to `covid_hosp` since state timeseries is now badly misleading. (not in this PR -- this is good to go as-is. just for future reference)

Reply:
`covid_hosp_state`, to distinguish from the facility-wise table, but yes